packages/sdk/src/controllers/VariableController.ts

```diff
@@ -694,6 +694,19 @@ export class VariableController {
             .then((result) => getEditorResponseData<null>(result));
     };
 
+    /**
+     * This method sets the context for an image variable
+     * @param id the id of the variable
+     * @param context the context record containing key-value pairs
+     * @returns
+     */
+    setImageVariableContext = async (id: string, context: Record<string, string | boolean>) => {
```

Comment:
should we use Dictionary here to be aligned with connectors
```suggestion
    setImageVariableContext = async (id: string, context: Dictionary) => {
```